### PR TITLE
Add Mariner image support in Kdump test cases

### PR DESCRIPTION
**Pull request by lubaihua33**

### CODE REVIEW

File: microsoft/testsuites/kdump/kdumpcrash.py [hunk 1]
```diff
@@ -307,6 +307,8 @@ def _kdump_test(self, node: Node, log_path: Path, log: Logger) -> None:
         # is not completed. So we can't use wait_tcp_port_ready function to judge if
         # the dump process is completed and the VM already boots up. We can use
         # try_connect function
+        # Wait for VM boot up and update IP address
+        time.sleep(30)
```
Comment (squirrelsc) [inline microsoft/testsuites/kdump/kdumpcrash.py]: Don't hard code sleep in logic. Use longer timeout instead.

Comment (lubaihua33) [reply microsoft/testsuites/kdump/kdumpcrash.py]: Have deleted this. I am thinking about a better way to enhance the kdump tests. I will post another PR.

**Comment by squirrelsc** [comment]
Please ask Lili to review and merge.

**Review by LiliDeng — APPROVED** [review]
I have tested some kdump cases against mariner, code looks good

**Pull request merged**